data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/buffer/KafkaBuffer.java

```diff
@@ -96,6 +96,11 @@ public Optional<Integer> getMaxRequestSize() {
         return Optional.of(producer.getMaxRequestSize());
     }
 
+    @Override
+    public Optional<Integer> getOptimalRequestSize() {
+        return Optional.of(producer.getMaxRequestSize() / 4);
```

Comment:
```suggestion
        return Optional.of(ONE_MEGABYTE);
```

Can we just make this value equal to 1MB? I think this is really what we are aiming for.

Reply:
I changed it to DEFAULT MAX REQUEST SIZE. not dividing with 4 anymore.